app/data/container-image-list.txt

```diff
@@ -14,3 +14,4 @@ cerc/fixturenet-eth-geth
 cerc/fixturenet-eth-lighthouse
 cerc/watcher-mobymask
 cerc/test-container
+cerc/eth_probe
```

Comment:
Can we be consistent within the file regarding dash or underscore?

Reply:
I can do that.  I had it matching the repo name, but it should work as a dash regardless of renaming the repo.